### PR TITLE
feat: rm npm2yarn add npm2yarn2pnpm plugin

### PR DESCRIPTION
**Pull request by ndom91**
## ☕️ Reasoning

Test: updated to [`npm2yarn2pnpm`](https://www.npmjs.com/package/@sapphire/docusaurus-plugin-npm2yarn2pnpm) remark plugin, instead of the `npm2yarn` one we had been using. Since `pnpm` has become the default package manager we use internally within NextAuth.js as well :+1: 

Doesn't seem to be working yet though.. :thinking: 

## 🧢 Checklist

- [x] Documentation
- [ ] Tests
- [ ] Ready to be merged

## 🎫 Affected issues

Please [scout and link issues](https://github.com/nextauthjs/next-auth/issues) that might be solved by this PR.

Fixes: INSERT_ISSUE_LINK_HERE

## 📌 Resources

- [Contributing guidelines](./CONTRIBUTING.md)
- [Code of conduct](./CODE_OF_CONDUCT.md)
- [Contributing to Open Source](https://kcd.im/pull-request)

### CODE REVIEW

File: docs/package.json [hunk 1]
```diff
@@ -21,12 +21,14 @@
...
     "@docusaurus/preset-classic": "^2.0.0-beta.21",
-    "@docusaurus/remark-plugin-npm2yarn": "^2.0.0-beta.21",
     "@docusaurus/theme-common": "2.0.0-beta.21",
     "@mdx-js/react": "1.6.22",
+    "@sapphire/docusaurus-plugin-npm2yarn2pnpm": "^1.1.0",
     "classnames": "^2.3.1",
+    "i": "^0.3.7",
     "mdx-mermaid": "^1.2.2",
     "mermaid": "^9.0.1",
+    "npm": "^8.13.2",
```
Comment (balazsorban44) [inline docs/package.json:31]: I guess this was added by mistake?

Comment (ndom91) [reply docs/package.json:31]: Yeah there was clearly some `npm i...` mis print along the way haha. Just up a draft PR to see if it worked on preview deploy, cause it wasn't working locally. Very much just a little test still..

**Comment by ndom91** [comment]
Not sure why the huge `pnpm-lock.yaml` change. But the `npm2yarn2pnpm` thing is finally working. Had to be a named import that is passed to the `remarkPlugins` array :facepalm:

File: docs/docusaurus.config.js [hunk 2]
```diff
@@ -147,6 +149,7 @@ module.exports = {
     [
       "@docusaurus/preset-classic",
       {
+        debug: true,
```
Comment (ThangHuuVu) [inline docs/docusaurus.config.js:152]: This seems mistakenly added

File: docs/docusaurus.config.js [hunk 3]
```diff
@@ -155,9 +158,9 @@ module.exports = {
           showLastUpdateAuthor: true,
           showLastUpdateTime: true,
           remarkPlugins: [
+            npm2yarn2pnpm,
```
Comment (ThangHuuVu) [inline docs/docusaurus.config.js:161]: Can we do `require('@sapphire/docusaurus-plugin-npm2yarn2pnpm')` here to make it consistent with the rest of the array? 🤔

Comment (ndom91) [reply docs/docusaurus.config.js:161]: No unfortunately not, that was the whole issue. It needs to be destructured. 

Maybe `require('@sapphire...').npm2yarn2pnpm` would work though? Idk

File: docs/docusaurus.config.js (hunk 2 above)
Comment (ndom91) [reply docs/docusaurus.config.js:152]: Not mistakenly added, but mistakenly not removed yet haha, lemme fix that.

Comment (ndom91) [reply docs/docusaurus.config.js]: Fixed

File: docs/docusaurus.config.js (hunk 3 above)
Comment (ndom91) [reply docs/docusaurus.config.js]: `require('@sapphire...').npm2yarn2pnpm` did seem to work :tada:

**Comment by balazsorban44** [comment]
Nice! One issue is that there is a blink when reloading the page:

image

Not sure if we can control this though, so it's fine for now. :+1:

Could we find something for switching TypeScript/JavaScript as well? :eyes:

**Review by balazsorban44 — APPROVED** [review]
Thanks!

**Pull request merged**